vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -86,94 +88,105 @@ Use `ent help` to review the list of available commands.
 
 ### Prepare and Publish a Bundle
-Use the publication system (pbs) to assemble your Entando project into a bundle that can be loaded into Kubernetes. You'll need your github credentials, a github repository to hold your bundle artifacts, and a Docker Hub account or organization.
+Use the publication system (pbs) to assemble your Entando project into a bundle that can be loaded into Kubernetes. You'll need your Github credentials, an empty GitHub repository to hold your bundle artifacts and a Docker Hub account or organization.
 1. Initialize the bundle directory
 ``` sh
 ent prj pbs-init
 ```
-2. Publish the build artifacts to github and Docker Hub
+2. Publish the build artifacts to Github and Docker Hub
 ``` sh
 ent prj pbs-publish
 ```
-3. Deploy the bundle into the Entando Component Repository.
+3. Deploy the bundle into the Entando Component Repository
 ``` sh
 ent prj deploy
 ```
 See [this tutorial](../../tutorials/ecr/publish-project-bundle.md) for more details.
 
-### Install the bundle into an application
-The ent CLI allows you to install a bundle without the need to access the Entando App Builder.
-Note: To install a given bundle, you need to be sure it has been deployed first.
-1. In your project folder run the following command
+### Install the bundle into an Application
+The CLI allows you to install a bundle without the need to access the Entando App Builder.
+
+Note: A bundle must be deployed before it can be installed.
+
+1. Run the following command from the project folder
 ``` sh
 ent prj install
 ```
-2. If you already installed the bundle, you can use `--conflict-strategy` to adopt a strategy for existing components (CREATE, SKIP, OVERRIDE)
+2. If a bundle has already been installed, use `--conflict-strategy` to adopt a strategy for existing components (CREATE, SKIP, OVERRIDE)
 ``` sh
 ent prj install --conflict-strategy=OVERRIDE
 ```
 
-### Run a Project locally
-1. Startup Keycloak. This uses docker-compose under the hood.
+### Run a Project Locally
+1. Initialize Keycloak, which leverages docker-compose
 ``` sh
 ent prj ext-keycloak start
 ```
-2. Startup the backend microservices
+2. Initialize backend microservices
 ``` sh
 ent prj be-test-run
 ```
-3. Startup one or more of the frontend widgets, each from its own shell.
+3. Initialize one or more frontend widgets, each from its own shell
 ``` sh
 ent prj fe-test-run
 ```
 
 See [this tutorial](../../tutorials/backend-developers/run-local.md) for more details.
 
-Alternatively, you can perform a completely clean install of the CLI by removing your `~/.entando` directory and then reinstalling the CLI per the instructions above. This will also remove the private copies of JHipster, Entando Blueprint, etc.
+Alternatively, removing the `~/.entando` directory and then reinstalling the CLI per the instructions above will perform a completely clean install. This will also remove the private copies of JHipster, Entando Blueprint, etc.
```

Comment:
minor, do you need 'completely', it seems implied.

Reply:
@jyunmitch just reminded me that this bit really confused me! agreed. @nshaw i wonder what the motivation for this comment is, let alone the emphasis on a clean install??

Reply:
@Lyd1aCla1r3 , hmm, it's kind of a clue to an advanced user on where the entando cli is installed. If you use similar tools, or if you have used Entando pre-CLI, it helps to explain why you don't need to do certain things any more, like install jhipster, etc. via npm. What's especially is the placement on the page. It should really be up under "Update the CLI" since it's an alternative to using the --update either. You could either update the CLI in place or remove .entando and start clean. I'd suggest moving it up to that section and keep the language as-is.